### ISSUE TITLE
Problem: Axis regularity in perlin noise

### ISSUE THREAD

**Issue by andrewq-qiu** [dilevin/computer-graphics-shader-pipeline#57]
My perlin noise seems to give axis-aligned effects with higher values of $k$ in $perlin(k * vec)$. Is this something else other people are running into? I can't seem to replicate anything similar to the bump examples provided

Here's one of my bump maps where $h = 0.1 * perlin(10 * s)$
![msrdc_0E5tdEe4Ii](https://github.com/dilevin/computer-graphics-shader-pipeline/assets/35282796/348680fe-88ec-4c25-8660-1f92cdc928d5)

**Comment by wenzhi-guo** [comment]
So if you see "axis-aligned" effects above, it usually means your noise function is not smooth. You mentioned you only see this effect for Perlin Noise with higher frequency?

**Comment by andrewq-qiu** [comment]
Yes, at lower frequencies it's blobbier but it still has some effects:

![image](https://github.com/dilevin/computer-graphics-shader-pipeline/assets/35282796/b56937ac-34ab-4e9c-89a8-4dc390175032)

This is with $perlin(s)$
